### PR TITLE
Refactored Classes to yaml with summer 2 dates

### PR DESCRIPTION
**Pull request by pcast01**

### CODE REVIEW

File: src/template/blog.js [hunk 1]
```diff
@@ -1,92 +0,0 @@
-import React, { useEffect, useState } from "react"
```
Comment (fpigeonjr) [inline src/template/blog.js:1]: did we mean to remove the blog @pcast01?

**Review by fpigeonjr — COMMENTED** [review]
wondering if we meant to remove the blog and jobs portion of career page?

**Comment by pcast01** [comment]
Well I did mean to get rid of the blog but I don't remember them even having a blog...

**Comment by fpigeonjr** [comment]
> Well I did mean to get rid of the blog but I don't remember them even having a blog...

Yes it was on the [career page](https://cbet.edu/careers). They use this page to announce job openings and had several blog posts listed here. This is the frontend to the admin area we created for them.

🤔 Think we should bring it back.

**Comment by pcast01** [comment]
Is there another way we can do the blogs, jobs for a new backend that costs
nothing??

**Comment by fpigeonjr** [comment]
> Is there another way we can do the blogs, jobs for a new backend that costs nothing??

Not that I am aware of. We will need to look into this but in the time being we should bring back the functionality they had.

**Comment by pcast01** [comment]
Yeah I agree. I have already put it back in a new branch. About to PR it.

**Pull request merged**